app/models/game.rb

```diff
@@ -3,7 +3,7 @@ class Game < ActiveRecord::Base
   belongs_to :deck
   has_many :guesses
 
-
+  #ZM: SEE PR 
```

Comment:
in def guessed_cards... can't you just pluck the :card rather then map the card?